File: Beecrowd/2764 Entrada e Saída de Data/mainv2.py
dia, mes, ano = input().split("/")

print("%02d/%02d/%02d" %(int(mes), int(dia), int(ano)))
print("%02d/%02d/%02d" %(int(ano), int(mes), int(dia)))
print("%02d-%02d-%02d" %(int(dia), int(mes), int(ano)))

File: Beecrowd/2764 Entrada e Saída de Data/mainv2.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main(){
    int dia, mes, ano;

    scanf("%d/%d/%d", &dia, &mes, &ano);

    cout << setfill('0') << setw(2) << mes << "/" << setw(2) << dia << "/" << setw(2) << ano << endl;
    cout << setw(2) << ano << "/" << setw(2) << mes << "/" << setw(2) << dia << endl;
    cout << setw(2) << dia << "-" << setw(2) << mes << "-" << setw(2) << ano << endl;

    return 0;
}